src/main/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/domain/entity/ElspPlanEntity.kt

```diff
@@ -61,7 +61,7 @@ data class ElspPlanEntity(
   @Column
   var updatedAtPrison: String,
 
-  @OneToMany(mappedBy = "elspPlan", cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.EAGER)
+  @OneToMany(mappedBy = "elspPlan", cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.LAZY)
```

Comment:
Didn't like having an EAGER fetch in here - feels like adding EAGER is a hack to get around some other problem in the code. So have taken these out.